@xen-orchestra/lite/src/components/ui/UiLegendProgressBar.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="legend">
+    <span class="circle" />
+    <slot name="label" />
+    <UiBadge class="badge"><slot name="value" /></UiBadge>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import UiBadge from "@/components/ui/UiBadge.vue";
+</script>
```

Comment:
I suggest being able to pass values as props in addition to slots.

This add flexibility. Props can be used for simple values, while the slots can be used for more complex value (custom component, HTML...)

```suggestion
<template>
  <div class="legend">
    <span class="circle" />
    <slot name="label">{{ label }}</slot>
    <UiBadge class="badge">
      <slot name="value">{{ value }}</slot>
    </UiBadge>
  </div>
</template>

<script lang="ts" setup>
import UiBadge from "@/components/ui/UiBadge.vue";

defineProps<{
  label?: string;
  value?: string;
}>();
</script>
```